dockerfiles/alpine_3.5_2.x

```diff
@@ -195,7 +195,8 @@ RUN set -x \
     && : "---------- luarocks ----------" \
     && cd / \
     && : "ldoc" \
-    && ${ROCKS_INSTALLER} install ldoc --server=http://rocks.moonscript.org \
+    && ( ${ROCKS_INSTALLER} install ldoc \
```

Comment:
You are changing behavior. Before: we will try to install "ldoc" only from the http://rocks.moonscript.org server. Now: we will try to install "ldoc" from the "default" server (from the config file) and only in case of failure from http://rocks.moonscript.org. is it ok?

Reply:
Decided to remove the patch from patch set.

Reply:
Patch is back)